src/bartender/filesystems/abstract.py

```diff
@@ -14,27 +14,32 @@ def localize_description(
     ) -> JobDescription:
         """Make given job description local to this file system.
 
-        :param description: A job description.
-        :param entry: The path to replace with the entry path of this file system.
-            For example given a file system with entry path of /remote/jobs
-            and given job_dir in job description of /local/jobs/myjobid
-            and given entry of /local/jobs
-            will return description with job dir /remote/jobs/myjobid .
-        :return: A job description local to this file system.
-        """  # noqa: DAR202
+        Args:
+            description: A job description.
+            entry: The path to replace with the entry path of this file
```

Comment:
The lines look shorter then the have to be.
Docconvert has a default max_line_length of 72, while flake8 uses 88. 

Can we make use of the full 88 chars per line? 
See https://docconvert.readthedocs.io/en/latest/intro.html#max-line-length

Reply:
That doesn't seem to work after having applied it with the default of 72 already... I went over it manually, hope I caught all.